src/finfield.h

```diff
@@ -128,8 +132,8 @@ extern  Obj             SuccFF;
 **  'TYPE_FF' returns the type of elements of the small finite field <ff>.
 **  'TYPE_FF0' returns the type of the zero of <ff>
 **
-**  Note that  'TYPE_FF' is a macro, so  do not call  it  with arguments that
-**  have side effects.
+**  Note that  'TYPE_FF' and TypeFF0 are macros, so  do not call them  
+**  with arguments that have side effects.
```

Comment:
Optional remark: They are macros which are used in precisely one place, inside function `TypeFFE` in `finfield.c`.  Moreover, `TypeFF` and `TypeFF0` are only used in that `.c` file, too Perhaps it would thus be best to remove them from the header, and turn the variables into static ones.